src/components/Checkbox.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+	CheckBox as GrommetCheckbox,
+	CheckBoxProps as GrommetCheckBoxProps,
+} from 'grommet';
+import * as React from 'react';
+import styled from 'styled-components';
+import { DefaultProps, Omit, RenditionSystemProps } from '../common-types';
+
+import asRendition from '../asRendition';
+
+/* Hover styles
+	1. Toggle (active & default)
+	2. Checkbox default
+	3. Checkbox active
+*/
+export const getHoverStyle = (props: any) => `
+	& label:hover input:not([disabled]) + span > span {
+		border-color: ${props.theme.colors.primary.dark};
+		background: ${props.theme.colors.primary.dark};
+	}
+	& label:hover input:not([disabled]) + div {
+		border-color: ${props.theme.colors.tertiary.main}!important;
+	}
+	& label:hover input[checked]:not([disabled]) + div {
+		border-color: ${props.theme.colors.primary.dark}!important;
+		background: ${props.theme.colors.primary.dark};
+	}
+`;
+
+const getBaseStyle = (props: any) => `
+	& > label > span {
+		font-family: ${props.theme.font};
+	}
+	& label input[checked] + div {
+		border-color: ${props.theme.colors.primary.main};
+		background: ${props.theme.colors.primary.main};
+		> svg {
+			stroke: #fff;
+		}
+	}
+	& label[disabled] {
```

Comment:
If I remember correctly, this can be set through the theme (not sure if it applies to the checkbox).

Reply:
Unfortunately not 

- https://github.com/grommet/grommet/blob/master/src/js/components/CheckBox/StyledCheckBox.js#L13
- https://github.com/grommet/grommet/blob/master/src/js/components/CheckBox/StyledCheckBox.js#L13

Althought I don't see anything in the radio button types either.

Reply:
There is opacity in the `global` style, doesn't it apply to the checkbox? Ah ok I see I also used the same CSS, maybe it doesn't apply to these, otherwise there is a prop in `global.control.disabled` in the theme. Sorry for the noise.